tests/ClientBuilderTest.php

```diff
@@ -210,6 +210,38 @@ public function optionsDataProvider()
             ['setErrorTypes', 0],
         ];
     }
+
+    public function testCompressionPluginAdded()
+    {
+        $builder = ClientBuilder::create(['enableCompression' => true, 'dsn' => 'https://completelyrandom@dsn.asdf/42']);
```

Comment:
Not that it changes something or is important at all, but we used through the codebase this DSN for the tests: `http://public:secret@example.com/sentry/1`